index.html

```diff
@@ -309,6 +309,25 @@ <h2>Skills</h2>
 					</ul>
 				</div>
 			</section>
+			
+			<section class="contact_section">
+				<div class="contact_section_container">
+					<h2>Contact me</h2>		
+					<div class="contact_text">
+						<p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+					</div>
+					<form autocomplete="on">						
+							<input type="text" id="name" name="name" placeholder="Name"><br>						
```

Comment:
I suggest adding the attributes to ensure that `Name field accepts up to 30 chars.` and `A user cannot submit the form if one or more of the fields is empty.`. As per the As per the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). Read about the `required and maxlength` attributes